universe/Conditions.cpp

```diff
@@ -11022,6 +11022,8 @@ void And::Eval(const ScriptingContext& parent_context, ObjectSet& matches,
         }
     }
 
+    // TODO: Enable if TraceLogger is needed
```

Comment:
`ObjList` is used primarily in the `TraceLogger` calls and is not needed otherwise.